### ISSUE TITLE
[FEATURE] Custom fields on Conversations

### ISSUE THREAD

**Issue by AD94-0** [leviFrosty/JW-Time#171]
Hey brother this may be to much to ask but I was thinking that something that would be extremely helpful is to add the ability to pick a specific publication that was used in the conversation. That way the app can keep a track of which publications have been used the most or have lead to a bible study. And this will help publishers be more productive and precise. Have like a stat record section or tab.

**Comment by leviFrosty** [comment]
Hi @AD94-0,

Thank you for this feature request. I would consider this to be a large feature request - and one that I'm not sure fits into the scope of WitnessWork.

This feature will be something I will mull over, I'm not 100% on how or if I'd like to implement this in its entirety.

However, I will consider adding a custom field to a conversation that allows you to notate something. For example, you can create a new custom field like `Publication: Enjoy Life Forever`. This will allow you to effectively have what you're requesting but also open the opportunity for other fields in the future as people see fit.

I hope this makes sense. Thank you for enjoying WitnessWork!

Yours,
Levi

**Comment by AD94-0** [comment]
Yeah I think that even better leaves the door open to endless categories to
track. I had a question though, I can I change my app language to Spanish

**Comment by leviFrosty** [comment]
I am currently pushing an update that will allow you to manually select your language and override your device settings. Expect this in the coming days.

**Comment by AD94-0** [comment]
Okay, thank you. Another question though is there a way to delete or modify
a time category?

**Comment by leviFrosty** [comment]
Yes - there was a bug with deleting categories. That bug is fixed in the upcoming release - please standby. Thank you!
